<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDCleaningWindow</class>
 <widget class="QMainWindow" name="WBSDCleaningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cleaning moment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item row="9" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="flWednesday"/>
     </item>
     <item row="9" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="clSunday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="flLabTuesday">
       <property name="text">
        <string>Flushing on Tuesday</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Flushing mode</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="flushMode"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="clLabWednesday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Wednesday</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QComboBox" name="flThursday"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="clLabSaturday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Saturday</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="flLabSaturday">
       <property name="text">
        <string>Flushing on Saturday</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="clLabMonday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Monday</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="clTuesday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QComboBox" name="flFriday"/>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="flSunday"/>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="clSaturday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cleanMode"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="clLabThursday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Thursday</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="clFriday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="flLabThursday">
       <property name="text">
        <string>Flushing on Thursday</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="flLabWednesday">
       <property name="text">
        <string>Flushing on Wednesday</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="clThursday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="flTuesday"/>
     </item>
     <item row="17" column="1">
      <widget class="QComboBox" name="flSaturday"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="clLabSunday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Sunday</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="flLabMonday">
       <property name="text">
        <string>Flushing on Monday</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="clLabFriday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Friday</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="clWednesday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="clLabTuesday">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cleaning on Tuesday</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="flLabFriday">
       <property name="text">
        <string>Flushing on Friday</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="clMonday">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="flLabSunday">
       <property name="text">
        <string>Flushing on Sunday</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="flMonday"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cleanLabMode">
       <property name="text">
        <string>Cleaning mode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cleanMode</tabstop>
  <tabstop>clSunday</tabstop>
  <tabstop>clMonday</tabstop>
  <tabstop>clTuesday</tabstop>
  <tabstop>clWednesday</tabstop>
  <tabstop>clThursday</tabstop>
  <tabstop>clFriday</tabstop>
  <tabstop>clSaturday</tabstop>
  <tabstop>flushMode</tabstop>
  <tabstop>flSunday</tabstop>
  <tabstop>flMonday</tabstop>
  <tabstop>flTuesday</tabstop>
  <tabstop>flWednesday</tabstop>
  <tabstop>flThursday</tabstop>
  <tabstop>flFriday</tabstop>
  <tabstop>flSaturday</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
